tests/test-cases/Group1-Docker-Commands/1-43-Docker-CP-Offline.robot

```diff
@@ -1,16 +1,16 @@
-# Copyright 2017 VMware, Inc. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#	http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License
+ Copyright 2017 VMware, Inc. All Rights Reserved.
+
```

Comment:
I think we have to have the copy right header commented out, otherwise it wouldn't pass the check.